### PR TITLE
Added detailed getting started instructions to README

### PR DESCRIPTION
**Pull request by KipHamiltons**

### CODE REVIEW

**Comment by ysims** [comment]
Maybe add in Windows instructions too

**Comment by KipHamiltons** [comment]
> Maybe add in Windows instructions too

@danielginn did you get this going/could you add these please?

**Comment by JosephusPaye** [comment]
Updated instructions, with Windows too.

File: README.md [hunk 1]
```diff
@@ -2,34 +2,81 @@
...
+<summary>Ubuntu installation instructions</summary>
+
+1. Install Git by running the following commands:
+
+```
+sudo apt update
+sudo apt install git
+```
+
+2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) to (pick the latest version).
```
Comment (Bidski) [inline README.md:25]:
```suggestion
2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions).
```

Comment (JosephusPaye) [reply README.md:25]: 😄 I probably meant to say pick the latest LTS version.

**Pull request merged**